test/hotspot/gtest/utilities/test_ostream.cpp

```diff
@@ -50,6 +51,32 @@ static void test_stringStream_is_zero_terminated(const stringStream* ss) {
   ASSERT_EQ(ss->base()[ss->size()], '\0');
 }
 
+static size_t count_char(const char* s, size_t len, char ch) {
+  size_t cnt = 0;
+
+  for (size_t i = 0; i < len; ++i) {
+    if (s[i] == ch) {
+      cnt++;
+    }
+  }
+  return cnt;
+}
+
+static size_t count_char(const stringStream* ss, char ch) {
+  return count_char(ss->as_string(), ss->size(), ch);
+}
+
+static void test_stringStream_tr_delete(stringStream* ss) {
```

Comment:
I think this is a unit test for `StringUtils::replace_no_expand`. It checks that the function can be used to remove substrings. There is no dependency on `stringStream`. Any string can be used.
Could you please move the test to `test_stringUtils.cpp`?

Reply:
yes, let me move it to test_stringUtils.cpp.

Reply:
I would like to keep stringStream because I think it's good idea to test the similar scenario. 
it's also handy to do memory management.